### DISCUSSION TITLE
The CTkScrollableFrame is not working properly for me

### DISCUSSION THREAD

**Discussion by nikita276956** [TomSchimansky/CustomTkinter#1261] [General]
I have tried to make a scrollable frame for my chatroom i have tried this code (posted a file)
and the messages are not being shown on the scrollable frame.
I also cant even move the scrollbar.
If I'll be able to make the messages appear on the scrollable frame it will be easier for me to use customtkinter and connect that example to my main app.
Also can i make a picture background for a scrollable frame?
(The code is attached below)
[global_chat - Copy.txt](https://github.com/TomSchimansky/CustomTkinter/files/10788196/global_chat.-.Copy.txt)

**Comment by jh-jensen** [comment]
Read from preious posts that .place() might make it not work, also it is advised to use either .grid() or .pack() instead unless in very specific cases. You could try with that s a start.

**Reply by nikita276956** [reply]
But in a normal CTkFrame it works fine, why at the scrollable frame its not?

**Reply by ElectricCandlelight** [reply]
Not looked into it but I'm guessing it's to do with how place works since it normally uses absolute positioning. 

Generally using place isn't a good idea unless you really have to.

**Reply by nikita276956** [reply]
Ohh i think i got it

**Reply by ElectricCandlelight** [reply]
If you could condense your code down to show what the problem is and post it here using a code block like ```example code block ``` 

Have you tried not using place and instead using grid which is more flexible and makes for a better reactive UI?

**Reply by nikita276956** [reply]
Yeah it was a problem with place, i used grid and now its working.
The only problem now is when i try to scroll down to the lastest messages
it does not show me them, instead only after a few messages it being
shown.. (the last messages that are out the user's screen)

**Reply by nikita276956** [reply]
as i found the scrollbar isnt shrinking when i am adding new messages please help me, i dont know what to do

**Comment by nikita276956** [comment]
Yeah it was a problem with place, i used grid and now its working.
The only problem now is when i try to scroll down to the lastest messages
it does not show me them, instead only after a few messages it being
shown.. (the last messages that are out the user's screen)

בתאריך יום ו׳, 24 בפבר׳ 2023, 00:39, מאת ElectricCandlelight ‏<
***@***.***>:
